examples/connext_dds/builtin_topics/c++11/application.hpp

```diff
@@ -36,7 +36,7 @@ inline void setup_signal_handlers()
 
 enum class ParseReturn { ok, failure, exit };
 
-enum class Entity { Publisher, Subscriber };
+enum class ApplicationKind { Publisher, Subscriber };
```

Comment:
```suggestion
enum class ApplicationKind { publisher, subscriber };
```

(Only types should have initial uppercase)